.gitignore

```diff
@@ -29,3 +29,5 @@ config
 configure
 libtool
 .dirstamp
+dispatch/module.modulemap
```

Comment:
these should start with /

else you will ignore any file named `dispatch/module.map` from the repository which is not what you mean here, see `gitignore(5)`:

```
       o   A leading slash matches the beginning of the pathname. For example,
           "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".
```